src/main/java/Entities/Song.java

```diff
@@ -8,7 +8,7 @@ public class Song {
 
     private final int id;
     private final String name;
-    private final List artistList;
+    private final List<String> artistList;
```

Comment:
Also I didn't know whether these should be ArtistUsers or Strings so let me know if changes need to be made on Monday

Reply:
I would assume its a list of ArtistUsers based off of the context, then you could just modify the getArtistList() method to go through the  artistList and retrieve the String representation of the artistUser's names and return a list of those